llmfoundry/__init__.py

```diff
@@ -46,6 +46,7 @@
     'MPTForCausalLM',
     'ComposerMPTCausalLM',
     'ComposerHFCausalLM',
+    'ComposerHFCausalLMFromPython',
```

Comment:
is this actually added anywhere?

Reply:
good catch; should be removed. this is an older implementation i had